pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1231,14 +1231,14 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
             doc = "On Shared and KeyShared subscriptions, if all available messages in the subscription are filtered "
                     + "out and not dispatched to any consumer, message dispatching will be rescheduled with a backoff "
                     + "delay. This parameter sets the initial backoff delay in milliseconds.")
-    private int dispatcherRetryBackoffInitialTimeInMs = 100;
+    private int dispatcherRetryBackoffInitialTimeInMs = 0;
```

Comment:
```suggestion
    private int dispatcherRetryBackoffInitialTimeInMs = 10;
```

Reply:
We shouldn't disable the limit. Setting it to a very low value like 10 ms or even 1 ms is better for saving CPU and preventing a spinning loop that burns a lot of CPU.